### PR TITLE
UI / UX changes in all in one dialog

### PR DESCRIPTION
**Pull request by mchf**
tuned_all-in-one_dialog
tuned_all-in-one_dialog_qt

done together with Ken
both reviewed by Ken

### CODE REVIEW

File: package/yast2-installation.changes [hunk 1]
```diff
@@ -1,18 +1,24 @@
 -------------------------------------------------------------------
+Thu Mar  2 13:33:41 UTC 2017 - mfilka@suse.com
+
+- adapted all-in-one dialog according comments from UX
```
Comment (mvidner) [inline package/yast2-installation.changes]: according **to** comments

File: package/yast2-installation.changes [hunk 2]
```diff
@@ -1,18 +1,24 @@
 -------------------------------------------------------------------
+Thu Mar  2 13:33:41 UTC 2017 - mfilka@suse.com
+
+- adapted all-in-one dialog according comments from UX 
+- 3.1.217.26
```
Comment (teclator) [inline package/yast2-installation.changes]: Fate or BSC number is missing

File: package/yast2-installation.changes (hunk 1 above)
Comment (mchf) [inline package/yast2-installation.changes]: yes, both ncurses and qt were checked together with ken

**Comment by mchf** [comment]
yes, both ncurses and qt were checked together with ken

**Comment by mchf** [comment]
updated with qt screenshot

**Comment by mchf** [comment]
Thanks for the reviews

**Pull request merged**